src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+    private static final String RESULT_SUPPLY = "supply";
```

Comment:
Pointless duplication, remove 
```
private static final String RESULT_SUPPLY = "supply";
private static final String RESULT_BUY = "buy";
```

Add constant for "comma" and all magic numbers, that you use